start.py

```diff
@@ -2,6 +2,7 @@
 # See the file 'LICENSE' for copying permission.
 
 import argparse
+import os
 import subprocess
 
 from dotenv import load_dotenv
```

Comment:
regarding this, I did not notice the problem that a previous PR you made introduced. I have described it [here](https://github.com/intelowlproject/IntelOwl/issues/876).

If you want to help regarding this, it would be really appreciated